src/Output/Writer.php

```diff
@@ -288,6 +303,42 @@ public function table(array $rows, array $styles = []): self
         return $this->colors($table);
     }
 
+    /**
+     * writes a key/value set to two columns in a row
+     *
+     * @example PHP Version ............................................................. 8.1.4
+     *
+     * @param string $first The text to write in left side
+     * @param string|null $second The text to write in right side
+     * @param array $options Options to use when writing Eg: ['fg' => Color::GREEB, 'bold' => 1, 'sep' => '-']
+     *
+     * @return self
+     */
+    public function twoColumnDetail(string $first, ?string $second = null, array $options = []): self
```

Comment:
i prefer `justify()`

Reply:
it's true that `justify` seems better to me. I'm not very good at giving names to things 😅